src/qibolab/instruments/qm/controller.py

```diff
@@ -23,7 +23,13 @@
 OCTAVE_ADDRESS_OFFSET = 11000
 """Offset to be added to Octave addresses, because they must be 11xxx, where
 xxx are the last three digits of the Octave IP address."""
-MAX_BATCH_SIZE = 30
+
+MAX_DURATION = int(4e4)
+"""Maximum duration of the control pulses [1q 40ns] [Rough estimate]."""
+MAX_READOUT = int(30)
+"""Maximum number of readout pulses [Not estimated]."""
+MAX_INSTRUCTIONS = int(1e6)
+"""Maximum instructions size [Not estimated]."""
```

Comment:
In another PR I was thinking to move these to `QMController` attributes, maybe with the same default values, so that it is possible to change them through the platform creation, without having to touch qibolab code. Another option is to use environment variables, but we would end up needed many of these (three for each instrument), so I think it is cleaner through the platform. 

It is important to have a flexible way to control them, that doesn't require creating a new branch (or installing in dev mode just for this), because most of these values are just estimates and we don't know the proper values yet.

@alecandido if you agree, I'll do it. Other than that this PR looks good to me. I would just like to test with QM before approving but that's not possible right now as the instruments are off.

Reply:
Agreed, being configurable through the Platform would be optimal. Possibly, these could even go in the instruments settings, and thus in the JSON (serialized from a `Bounds` instance).